mappings/net/minecraft/screen/ScreenHandlerSyncHandler.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_5916 net/minecraft/screen/ScreenHandlerSyncHandler
```

Comment:
Maybe just `ScreenHandlerSync`, two handlers in name sounds awkward

Reply:
I think ending with `Sync` feels more like an event rather than a continuous handler, I originally got suggested `ScreenHandlerSyncer` but found `Syncer` to not feel like a real word, is that potentially better?

Reply:
I wish I hadn't seen the mojmap name for this, or I would be suggesting better names for this. I can't get behind it, it's so awkward :/ But I suppose it's at least better than `class_5916`

Reply:
we can just use this name for now and update later